integration-test/cli.Tests.ps1

```diff
@@ -0,0 +1,178 @@
+# This file contains test cases for https://pester.dev/
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+. $PSScriptRoot/common.ps1
+
+Describe 'Console apps (<framework>) - normal build' -ForEach @(
+    @{ framework = "net7.0" }
+) {
+    BeforeAll {
+        DotnetNew 'console' 'console-app' $framework
+    }
+
+    BeforeEach {
+        Remove-Item "./console-app/bin/Release/$framework/*.src.zip" -ErrorAction SilentlyContinue
+    }
+
+    It "uploads symbols and sources" {
+        $result = RunDotnetWithSentryCLI 'build' 'console-app' $True $True $framework
+        $result.UploadedDebugFiles() | Sort-Object -Unique | Should -Be @('console-app.pdb')
+        $result.ScriptOutput | Should -AnyElementMatch 'Found 1 debug information file \(1 with embedded sources\)'
+        $result.ScriptOutput | Should -AnyElementMatch 'Resolved source code for 0 debug information files'
+    }
+
+    It "uploads symbols" {
+        $result = RunDotnetWithSentryCLI 'build' 'console-app' $True $False $framework
+        $result.UploadedDebugFiles() | Sort-Object -Unique | Should -Be @('console-app.pdb')
+        $result.ScriptOutput | Should -AnyElementMatch 'Found 1 debug information file \(1 with embedded sources\)'
+    }
+
+    It "uploads sources" {
+        $result = RunDotnetWithSentryCLI 'build' 'console-app' $False $True $framework
+        $result.ScriptOutput | Should -AnyElementMatch 'Skipping embedded source file: .*/console-app/Program.cs'
+        $result.UploadedDebugFiles() | Should -BeNullOrEmpty
+    }
+
+    It "uploads nothing when disabled" {
+        $result = RunDotnetWithSentryCLI 'build' 'console-app' $False $False $framework
+        $result.UploadedDebugFiles() | Should -BeNullOrEmpty
+    }
+}
+
+Describe 'Console apps (<framework>) - native AOT publish' -ForEach @(
+    @{ framework = "net7.0" },
+    @{ framework = "net8.0" }
```

Comment:
Adding a new framework will be done here?
Could you please write some doc/instruction on how this works/how to keep it maintained or add new frameworks?

Reply:
I've written some intro in CONTRIBUTING.md (https://github.com/getsentry/sentry-dotnet/pull/2781/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R138).

And you've guessed right so the question of adding frameworks should be clear then :) 

But if you have questions that you don't know the answer to (or anyone else does), feel free to post here and I'll update the docs.